src/utils/signature.ts

```diff
@@ -0,0 +1,25 @@
+import { verifyMessage } from '@ambire/signature-validator'
+import { JsonRpcSigner } from '@ethersproject/providers'
+import { readProvider } from 'constants/readProvider'
+
+export const verifyWallet = async (signer: JsonRpcSigner, message: string) => {
```

Comment:
**question** do we need this function? looks like it just wraps `signer.signMessage(message)`.

If so, perhaps a more representitive name might be `signMessage`